src/cmds/migration/run.ts

```diff
@@ -165,6 +182,17 @@ const runMigrationCommand: yargs.CommandModule = {
     },
 };
 
+export const getRange = (range: string): [number, number] | null => {
+    const match = range.match(/^([0-9]+):([0-9]+)$/);
+    if (!match) {
+        return null;
+    }
+    const from = Number(match[1]);
+    const to = Number(match[2]);
+
+    return from <= to ? [from, to] : null;
```

Comment:
I might return 
```
{
  from,
  to
}
```

object

Just for readability.

Reply:
I've introduced new IRange interface, thanks for the tip